app/javascript/widget/App.vue

```diff
@@ -130,10 +128,10 @@ export default {
     createWidgetEvents(message) {
       const { eventName } = message;
       const isWidgetTriggerEvent = eventName === 'webwidget.triggered';
-      if (isWidgetTriggerEvent && this.showUnreadView) {
+      this.setUserLastSeen();
+      if (isWidgetTriggerEvent) {
         return;
       }
-      this.setUserLastSeen();
```

Comment:
removed check for showUnreadView since that will always be false for us. `isWidgetTriggerEvent` will also always be true since I believe the only events that get sent here are `webwidget.triggered` events, but just left in in incase that ever changes